api/_globals/crypto.json

```diff
@@ -0,0 +1,108 @@
+{
+  "api": {
+    "crypto": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/crypto_property",
+        "spec_url": "https://w3c.github.io/webcrypto/#dom-windoworworkerglobalscope-crypto",
+        "support": {
+          "chrome": {
+            "version_added": "37"
+          },
+          "chrome_android": {
+            "version_added": "37"
+          },
+          "deno": {
+            "version_added": "1.0"
+          },
+          "edge": {
+            "version_added": "12"
+          },
+          "firefox": {
+            "version_added": "1"
```

Comment:
I didn't believe this, but I confirmed support in Firefox 4, so it's probably correct :)